src/NUnit3TestAdapterInstall/app.config

```diff
@@ -1,11 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<configuration>
-	<runtime>
-		<assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
-			<dependentAssembly>
-				<assemblyIdentity name="nunit.engine" publicKeyToken="2638cd05610744eb" culture="neutral"/>
-				<bindingRedirect oldVersion="0.0.0.0-3.0.5797.27553" newVersion="3.0.5797.27553"/>
-			</dependentAssembly>
-		</assemblyBinding>
-	</runtime>
-	<startup/></configuration>
```

Comment:
Since we're not compiling an assembly in this project, nor are we dealing with versions of the engine < 3.0.5797.